components/server/src/ome/services/ThumbnailBean.java

```diff
@@ -1248,9 +1244,10 @@ protected void actOnOneGroup(Set<Long> pixelsIds) {
             }
         }
 
-        // This will return null if a thumbnail is blocked waiting on import completion
+        thumbnailMetadata = _createThumbnail(thumbMetaData);
```

Comment:
This addition now throws an exception leading to the error in insight.
This will now be "covered" up by https://github.com/openmicroscopy/openmicroscopy/pull/5865